acm/templates/policies/application-policies.yaml

```diff
@@ -45,6 +45,9 @@ spec:
                       valueFiles:
                       - "/values-global.yaml"
                       - "/values-{{ .name }}.yaml"
+                      # We cannot use $.Values.global.clusterVersion because that gets resolved to the
+                      # hub's cluster version, whereas we want to include the spoke cluster version
+                      - '/values-{{ `{{ (cat (semver (lookup "operator.openshift.io/v1" "OpenShiftControllerManager" "" "cluster").status.version).Major (semver (lookup "operator.openshift.io/v1" "OpenShiftControllerManager" "" "cluster").status.version).Minor) | replace " " "." }}` }}-{{ .name }}.yaml'
```

Comment:
maybe s/cat/print/ ?

Reply:
Do ACM templates have a print function though? I don't see it mentioned here https://github.com/stolostron/go-template-utils nor here https://github.com/stolostron/go-template-utils/commit/5466c772c9e434611156298a80f4c49d7a56bd96#diff-599bd7869ad64965d7a512c105d93e3265bdc6150b13f127c8417ced8bf179b1

Reply:
Ok so there is a printf function and I tested a new variant. Thanks for pointing it out!